.github/workflows/rust.yml

```diff
@@ -44,9 +44,14 @@ jobs:
         sudo apt-get update
         sudo apt-get install qemu-system-x86 ovmf -y
 
-    - name: Build
+    - name: Build (without unstable)
       run: cargo xtask build --target x86_64
 
+      # It is sufficient if we include the "--include-unstable" flag only for one build as we
+      # only plan to have platform-independent functionality behind basic feature gates.
+    - name: Build (with unstable)
+      run: cargo xtask build --target x86_64 --include-unstable
```

Comment:
This job can be dropped, the build variations are covered by https://github.com/rust-osdev/uefi-rs/pull/590. (The new test-without-unstable job is still needed though.)

Reply:
The problem is, that the unit test for the make_boxed utility has different behavior with and without the unstable feature. I add a comment to the CI step to make it more clear.